Assets/Scripts/Player/PlayerControler.cs

```diff
@@ -9,6 +9,8 @@ public class PlayerControler : MonoBehaviour
 {
     public PlayerState playerState { get; private set; }
 
+    [SerializeField] public float hitPoints;
```

Comment:
czy to jest jakieś życie gracza?

Reply:
yup, wykorzystywane do skryptu zadającego dmg i triggerującego śmierc gracza